index.html

```diff
@@ -1127,6 +1163,150 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
             </div>
           </section>
 
+          <section id="reading-writing-profiles" inlist="" rel="schema:hasPart" resource="#reading-writing-profiles">
+            <h2 property="schema:name">Reading and Writing Profiles</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+                <section id="reading-profile" inlist="" rel="schema:hasPart" resource="#reading-profile">
+                    <h3 property="schema:name">Reading Profile</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                        <p about="" id="reader-application-webid-profile" rel="spec:requirement"
+                            resource="reader-application-webid-profile">
+                            <span property="spec:statement">To retrieve a representation of a Solid WebID Profile, <a
+                                    rel="spec:requirementSubject" href="#ReaderApplication">Reader Application</a> <span
+                                    rel="spec:requirementLevel" resource="spec:MUST">MUST</span> MUST make
+                                an <code>HTTP GET</code> request targeting a Solid WebID Profile resource including a
+                                <code>Content-Type header</code> requesting a representation in <code>text/turtle</code> or
+                                <code>application/ld+json.</code></span>
+                        </p>
+                        <section id="webid-profile-data-model" inlist="" rel="schema:hasPart"
+                            resource="#webid-profile-data-model">
+                            <h3 property="schema:name" content="WebID Profile Data Model">Solid WebID Profile</h3>
+                            <div datatype="rdf:HTML" property="schema:description">
+                                <p>The <a href="#solid-profile">Solid WebID Profile</a> has the following properties:</p>
+                                <ul>
+                                    <li>One <code>rdf:type</code> property whose object is <code>foaf:Agent</code>.</li>
+                                    <li>One <code>space:preferencesFile</code>.</li>
+                                    <li>Zero or more <code>ldp:inbox</code>.</li>
+                                    <li>Zero, one, or more <code>space:storage</code>.</li>
+                                    <li>Zero, one, or more <code>rdfs:seeAlso</code> to refer to related <a
```

Comment:
"one" is redundant - "Zero, or more" is the same thing

Reply:
```suggestion
                                    <li>Zero or more <code>space:storage</code>.</li>
                                    <li>Zero or more <code>rdfs:seeAlso</code> to refer to related <a
```